hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -305,6 +301,247 @@ async def attempts(self):
+            body = await resp.json()
+            for job_group in body['job_groups']:
+                yield JobGroup.submitted_job_group(self._batch, job_group['job_group_id'], _last_known_status=job_group)
+            last_job_group_id = body.get('last_job_group_id')
+            if last_job_group_id is None:
+                break
+
+    async def jobs(
+        self,
+        q: Optional[str] = None,
+        version: Optional[int] = None,
+        recursive: bool = False,
+    ) -> AsyncIterator[JobListEntryV1Alpha]:
+        self._raise_if_not_submitted()
+        if version is None:
+            version = 1
+        last_job_id = None
+        while True:
+            params: Dict[str, Any] = {'recursive': str(recursive)}
+            if q is not None:
+                params['q'] = q
+            if last_job_id is not None:
+                params['last_job_id'] = last_job_id
+            resp = await self._client._get(
+                f'/api/v{version}alpha/batches/{self.batch_id}/job-groups/{self.job_group_id}/jobs', params=params
+            )
+            body = cast(GetJobsResponseV1Alpha, await resp.json())
+            for job in body['jobs']:
+                yield job
+            last_job_id = body.get('last_job_id')
+            if last_job_id is None:
+                break
+
+    async def status(self) -> GetJobGroupResponseV1Alpha:
+        self._raise_if_not_submitted()
+        resp = await self._client._get(f'/api/v1alpha/batches/{self.batch_id}/job-groups/{self.job_group_id}')
+        json_status = await resp.json()
+        assert isinstance(json_status, dict), json_status
+        self._last_known_status = cast(GetJobGroupResponseV1Alpha, json_status)
+        return self._last_known_status
+
+    async def last_known_status(self) -> GetJobGroupResponseV1Alpha:
+        self._raise_if_not_submitted()
+        if self._last_known_status is None:
+            return await self.status()  # updates _last_known_status
+        return self._last_known_status
+
+    def create_job(self, image: str, command: List[str], **kwargs) -> Job:
+        return self._batch._create_job(self, {'command': command, 'image': image, 'type': 'docker'}, **kwargs)
+
+    def create_jvm_job(self, jar_spec: Dict[str, str], argv: List[str], *, profile: bool = False, **kwargs):
+        return self._batch._create_job(
+            self, {'type': 'jvm', 'jar_spec': jar_spec, 'command': argv, 'profile': profile}, **kwargs
+        )
+
+    def create_job_group(
+        self,
+        *,
+        attributes: Optional[Dict[str, str]] = None,
+        callback: Optional[str] = None,
+        cancel_after_n_failures: Optional[int] = None,
+    ) -> 'JobGroup':
+        return self._batch._create_job_group(
+            self,
+            attributes=attributes,
+            callback=callback,
+            cancel_after_n_failures=cancel_after_n_failures,
+        )
+
+    # FIXME Error if this is called while in a job within the same job group
```

Comment:
Resolve the FIXME or create an issue.

Reply:
https://github.com/hail-is/hail/issues/14338